tests/test_unittest/test_storage.py

```diff
@@ -944,7 +944,7 @@ def test_sum_gpu():
         shape=shape,
     )
 
-    q1[i1 : i2 + 1, jslice, 0] = cp.sum(q2[i1 : i2 + 1, jslice, :], axis=2)
+    q1[i1 : i2 + 1, jslice, 0] = cp.sum(q2.data[i1 : i2 + 1, jslice, :], axis=2)
```

Comment:
Looks like this test is supposed testing suming over storages (which is a ufunc) if we just use .data this test tests nothing - i.e. we should probably remove the test as it's just no longer supported to do sums.